### PR TITLE
betterdisplay: rename from betterdummy

### PR DESCRIPTION
**Pull request by chenrui333**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/b/betterdisplay.rb [hunk 1]
```diff
@@ -1,40 +1,33 @@
...
   ]
+
+  caveats do
+    discontinued
+
+    <<~EOS
+      #{token} has been renamed 'betterdisplay',
+      install betterdisplay to continue receiving updates;
+
+        brew install --cask betterdisplay
```
Comment (bevanjkay) [inline Casks/b/betterdisplay.rb:30]: @chenrui333 We can remove the caveat now

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/b/betterdisplay.rb (hunk 1 above)
Comment (chenrui333) [reply Casks/b/betterdisplay.rb:30]: oh true

File: Casks/b/betterdisplay.rb [hunk 2]
```diff
@@ -1,40 +1,22 @@
...
   zap trash: [
-    "~/Library/Application Support/BetterDummy",
-    "~/Library/Caches/me.waydabber.BetterDummy",
-    "~/Library/HTTPStorages/me.waydabber.BetterDummy",
-    "~/Library/Preferences/me.waydabber.BetterDummy.plist",
+    "~/Library/Application Scripts/me.waydabber.BetterDisplayHelper",
+    "~/Library/Application Scripts/me.waydabber.BetterDisplay",
+    "~/Library/Containers/me.waydabber.BetterDisplayHelper",
+    "~/Library/Containers/me.waydabber.BetterDisplay",
+    "~/Library/Preferences/me.waydabber.BetterDisplay.plist",
```
Comment (chenrui333) [inline Casks/b/betterdisplay.rb:20]: I actually could not find these resources

**Comment by chenrui333** [comment]
gonna drop this PR, did not realize there is already betterdisplay in place.

**Comment by chenrui333** [comment]
the zap setup is wrong with betterdisplay.

**Comment by bevanjkay** [comment]
@chenrui333 We should still remove `betterdummy` and update the `cask_renames` as the caveat was added before `cask_renames` were possible.

**Pull request closed without merging**